app/controllers/affiliates_controller.rb

```diff
@@ -5,14 +5,12 @@ def new
   end
 
   def create
-    attrs = params[:affiliate]
-
-    if attrs.blank?
+    if affiliate_params.blank?
```

Comment:
Do we think this is a legitimate code path? Could we get away with relying on the fact the `Affiliate` will be invalid if we're missing the params?